packages/cli/src/commands/secrets/index.js

```diff
@@ -130,7 +36,7 @@ const main = async client => {
     client.output.print(
       help(secretsCommand, { columns: client.stderr.columns })
     );
-    await exit(2);
```

Comment:
All our other commands use return values instead of `exit`, so migrated this to match. This is our final use of the `exit` library so that is removed in this PR as well.

Reply:
niceeeee